### PR TITLE
Creating new branch test

### PR DESCRIPTION
**Pull request by greavr**
This is awesome!

### CODE REVIEW

File: backend/code/index.py [hunk 1]
```diff
@@ -99,9 +99,6 @@ def ping():
 def default():
     return render_template('default.html')
 
-
-
-
```
Comment (lchow3) [inline backend/code/index.py:104]: What is this? Why did you do this?

**Review by lchow3 — CHANGES_REQUESTED** [review]
Don't fuck it up

File: backend/code/index.py (hunk 1 above)
Comment (greavr) [inline backend/code/index.py:104]: LGTM

**Review by lchow3 — APPROVED** [review]
LGTM

**Pull request merged**